app/models/document.rb

```diff
@@ -44,7 +44,7 @@ class Document < ApplicationRecord
   attr_accessor :sluggable_string
 
   def self.published
-    joins(:published_edition)
+    joins(:editions).where(editions: { state: "published" })
```

Comment:
I think this changes the behaviour. The previous `joins(:published_edition)` included both "published" and "withdrawn", where it's now limited just to "published" editions.

So I'm thinking we should probably rename this method to `self.live` and maintain the existing behaviour. Wdyt?